Lesson6/isdigit.py

```diff
@@ -0,0 +1,10 @@
+num = input()
```

Comment:
so.. if I pass  `-.777` - I get **You have written a positive decimal number: -.777!**, but in isnt positive

Reply:
Oh, right. I will fix this!